### DISCUSSION TITLE
custom settings and localization? Any best practices already?

### DISCUSSION THREAD

**Discussion by bashuss** [OrchardCMS/OrchardCore#14908] [Q&A]
I want to use custom settings to select or enter some standard content for the site, but this content needs to be different for each supported locale. What is the best way to approach this?

**Comment by hishamco** [comment]
We already have `OrchardCore.ContentLocalization` module that allows you to localize your content based on the current supported cultures, please refer to https://docs.orchardcore.net/en/latest/docs/reference/modules/ContentLocalization/

**Reply by bashuss** [reply]
I know the ContentLocalization. The question is about best practices. I did not manage to localize a Setting directly. I just end up with a Settings Page that allows me to select a locale. But there is no second Settings Page for the other locales. Am I supposed to look up the created Instance of the Settings and create a localized Version of it? Would I need the Settings Type to be creatable and listable to do that?
Or is it a better Idea to add a Bag to the Settings Type and add the localized Settings in different entries there?
Or should I create two different Setting Types for two locales and add another, if I need another locale in the future?

Am I missing the obivious easy way?

**Reply by hishamco** [reply]
Your question is clear now :)

If I'm not wrong the custom settings are a content type, so you can attach `LocalizationPart` to it

**Reply by bashuss** [reply]
yes, you can do that. But there is only one SettingsPage showing up for the ContentType, not one for each locale you configured for the website. So how am I supposed to work with that? 
Is this intentionally done like this? I does not look to me like it, as I do not see or understand a direct way to add localized versions of the setting.

**Reply by hishamco** [reply]
I'm not sure if it's by design or not

/cc @deanmarcussen @sebastienros

**Reply by bashuss** [reply]
Actually I retried: I added a localization part to my custom settings ContentType. Localization selector shows up on the Settings Page, but if I try to create another locale, I get a page not found error.

I guess this is a bug. -> https://github.com/OrchardCMS/OrchardCore/issues/15237
